src/app/books/containers/books-list/books-list.container.ts

```diff
@@ -44,7 +49,13 @@ export class BooksListContainer implements OnInit {
     this.queryParams.page = +data.get('page')!;
     this.queryParams.limit = +data.get('limit')!;
 
-    this.books$.next();
+    this._bookService.books$.next();
+  }
+
+  public filterPage(filters: any): void {
```

Comment:
@wis-zadonsky will be better to use `unknown` type instead of `any` and specify necessary type where the variable is used.